src/components/layout/Stack.js

```diff
@@ -0,0 +1,24 @@
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { system } from 'styled-system';
+import Grid from './Grid';
+
+export const Stack = styled(Grid)(
```

Comment:
We've talked about this yesterday but I don't think just renaming one prop is a good reason to create a new component — it'll be very hard to explain why it's necessary. Personally, I don't see a reason why this component exists.

Reply:
I do. It's the whole point of design system and a common language: designers don't thing in CSS grids. They think in stacks, tiles, columns. So adding higher level abstraction helps to establish a common language between eng and designers. I even think to rewrite the Stack component for the workshop using `* + *` to make it more rudimentary. I also think that our `Stack` does to much and should be better called `Grid`. The Stack should only stack children vertically. One additional prop could be added for the horizontal alignment but I don't think Stack should be exposing all Grid props anymore. So the point here: if you need to distribute children vertically, use Stack. If you need a more complex layout, use Grid.

Reply:
That's actually a good discussion to mention during the workshop. I'll add it to the materials.

Reply:
I agree with the second part of your monolog ;-) but this component doesn't show that. My point here is: think how you'd convince a participant why it's needed. With this example, it's going to be hard.

I'm also thinking about splitting my Stack component into Grid and Stack with the lobotomized owl — it's just easier to use.

Reply:
> think how you'd convince a participant why it's needed.

That's not the point of the workshop. It's to show how to deal with lower and higher level abstractions in different scenarios. This is just an example and in reality they will go with `Columns` or something else.

Reply:
I disagree with this. If they aren't convinced we're showing something useful (and with this example, they won't be), they won't get the idea.

Reply:
So what's your proposal?

Reply:
Merge as is and then we do the 🤪🦉

Reply:
I've redone it with the owl already. Check it ou thttps://github.com/component-driven/component-driven-development/pull/64/commits/5d3c181dee88ce8727bf19124546adf9ee71f2e7